pytorch_lightning/utilities/torch_distributed.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import pickle
+
+import torch
+
+from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_7
+
+log = logging.getLogger(__name__)
+
+if torch.distributed.is_available():
+    from torch.distributed import Backend, broadcast, get_backend, get_rank, GroupMember
+
+# The code underneath is taken from PyTorch ``torch/distributed/distributed_c10d.py``
+# and enable broadcasting for PyTorch 1.6 and lower.
```

Comment:
maybe this file should go to the "overrides" directory. 
could be good to have all customized pytorch code in one place.
not sure, let me know what you think.

Reply:
Makes sense. Let me do that :)